_pages/federal-fieldguide/doing-the-work.md

```diff
@@ -4,9 +4,311 @@ title: Federal field guide
 permalink: /federal-field-guide/doing-the-work/
 sidenav: federal
 sticky_sidenav: true
+subnav:
+- text: Kick-off meeting
+  href: '#host-an-effective-post-award-kick-off-meeting-to-energize-folks-for-the-work-to-come'
+- text: Agile oversight
+  href: '#oversee-agile-projects-by-measuring-end-user-outcomes-instead-of-requiring-project-teams-to-perform-tasks-by-specific-dates'
+- text: Post-award administration
+  href: '#post-award-contract-administration-looks-different-in-agile'
+- text: QASP monitoring
+  href: '#monitor-conformance-with-the-qasp-at-the-end-of-every-sprint'
+- text: Footnotes
+  href: '#footnotes'
 ---
 
 # Federal Field Guide
 {:.usa-sr-only}
 
 ## Doing the work
+The following sections identify challenges and strategies to reduce risk during the post-award phase of custom technology projects in government.
+
+### Host an effective post-award kick-off meeting to energize folks for the work to come.
+
+By **Stephanie Rivera**, **Miatta Myers**, **Vicki McFadden**, and **Mark Hopson**<br>
+{:.guide-authors}
+
+**Challenge:** Traditional post-award kick-off meetings usually do not adequately engage or prepare the contractor to begin work.
+{:.guide-download}
+
+#### Executive Summary
+A thoughtful post-award kick-off meeting should introduce new team members, empower the government product owner, plan next steps, and get the team excited about the upcoming work.
+
+#### Recommendations
+
+[FAR Part 42.5](https://www.acquisition.gov/content/part-42-contract-administration-and-audit-services#i1075565) encourages Contracting Officers (COs) to hold a post-award kick-off meeting with the winning contractor. That is often treated somewhere between a check-box exercise and an afterthought.
+
+We recommend being more intentional about the kick-offs, either as one combined meeting (contractual + project) or as two separate meetings scheduled in close proximity to one another. Some Contracting Officers have strong feelings about their procedures for a FAR-required kickoff, and prefer to hold their own separate meeting.
+
+Goals of a post-award kick-off:
+
```

Comment:
May make these into regular paragraphs instead of lists

Reply:
cool, they're rendered as paragraphs now - preview below 

![Screen Shot 2020-09-03 at 9 20 17 AM](https://user-images.githubusercontent.com/2374206/92120185-bbe6e400-edc6-11ea-8341-f918bdae5433.png)